src/com/organizationmanager/OrganizationManager.java

```diff
@@ -0,0 +1,344 @@
+        Integer annualTurnover = Integer.parseInt(scanner.nextLine());
+        System.out.print("Type (COMMERCIAL, PUBLIC, GOVERNMENT, TRUST, PRIVATE_LIMITED_COMPANY): ");
+        String typeStr = scanner.nextLine();
+        OrganizationType type = typeStr.isEmpty() ? null : OrganizationType.valueOf(typeStr);
+        System.out.print("Postal Address (zipCode): ");
+        String zipCode = scanner.nextLine();
+
+        Coordinates coordinates = new Coordinates(x, y);
+        Address postalAddress = new Address(zipCode);
+        Organization updatedOrg = new Organization(id, name, coordinates, annualTurnover, type, postalAddress);
+
+        organizations.remove(orgToUpdate);
+        organizations.add(updatedOrg);
+        System.out.println("Organization updated successfully.");
+    }
+
+    private void removeById(Long id) {
+        Organization orgToRemove = organizations.stream()
+                .filter(org -> org.getId().equals(id))
+                .findFirst()
+                .orElse(null);
+
+        if (orgToRemove == null) {
+            System.out.println("Error: Organization with ID " + id + " not found.");
+        } else {
+            organizations.remove(orgToRemove);
+            System.out.println("Organization removed successfully.");
+        }
+    }
+
+    private void clearCollection() {
+        organizations.clear();
+        System.out.println("Collection cleared successfully.");
+    }
+
+    private void saveToFile(String fileName) {
+        try (FileWriter writer = new FileWriter(fileName)) {
+            writer.write("<organizations>\n");
+            for (Organization org : organizations) {
+                writer.write("  <organization>\n");
+                writer.write("    <id>" + org.getId() + "</id>\n");
+                writer.write("    <name>" + org.getName() + "</name>\n");
+                writer.write("    <coordinates>\n");
+                writer.write("      <x>" + org.getCoordinates().getX() + "</x>\n");
+                writer.write("      <y>" + org.getCoordinates().getY() + "</y>\n");
+                writer.write("    </coordinates>\n");
+                writer.write("    <creationDate>" + org.getCreationDate() + "</creationDate>\n");
+                writer.write("    <annualTurnover>" + org.getAnnualTurnover() + "</annualTurnover>\n");
+                writer.write("    <type>" + (org.getType() != null ? org.getType() : "") + "</type>\n");
+                writer.write("    <postalAddress>\n");
+                writer.write("      <zipCode>" + org.getPostalAddress().getZipCode() + "</zipCode>\n");
+                writer.write("    </postalAddress>\n");
+                writer.write("  </organization>\n");
+            }
+            writer.write("</organizations>\n");
+            System.out.println("Collection saved to " + fileName);
+        } catch (IOException e) {
+            System.err.println("Error saving to file: " + e.getMessage());
+        }
+    }

```

Comment:
why are you writing the string by hand? you laready used xml library in parsing the file